clarity.d.ts

```diff
@@ -303,6 +312,27 @@ interface IClarityAssertFailedEventState extends IInstrumentationEventState {
 interface IClarityDuplicatedEventState extends IInstrumentationEventState {
   currentImpressionId: string;
 }
+
+interface IShadowDomInconsistentEventState extends IInstrumentationEventState {
+  // JSON of node indicies, representing the DOM
+  dom: NumberJson;
```

Comment:
In a way..sending back the entire document here, so it is specific to a node - document node :) However, since we represent nodes with numeric indices it's ends up being JSON with numeric keys, hence the new type.

---
In reply to: [132251527](https://github.com/Microsoft/clarity-js/pull/25#discussion_r132251527) [](ancestors = 132251527)